scripts/install-dev.sh

```diff
@@ -146,13 +146,23 @@ show_important_note() {
 
 has_python() {
   "$1" -c '' >/dev/null 2>&1
-  if [ "$?" -eq 127 ]; then
+  if [ "$?" -ne 0 ]; then
     echo 0
   else
     echo 1
   fi
 }
 
+install_indygreg_python() {
```

Comment:
We appreciate the hard work by indygreg, but it's not necessary to name our functions and variables following him. :wink:
Let's just use the purpose-oriented naming like "boostrapping", "relocatable", "portable", "static", etc.